src/pages/Loans/LoansOverview/components/LoanActionInfo/LendGroup.tsx

```diff
@@ -0,0 +1,39 @@
+import { LoanAsset } from '@interlay/interbtc-api';
```

Comment:
`BorrowGroup` and `LendGroup` could be merged into one general component and we could pass `apy` and `reward` props directly without need to specify if it's lend or borrow.